lib/deck.rb

```diff
@@ -0,0 +1,22 @@
+class Deck
+    attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    cards.count
+  end
+
+  def cards_in_category(category)
+    cards.find_all do |card|
+        card.category == category
+        end
+    end
```

Comment:
Nice use of `find_all`. Once again, reference instance variables when you can instead of their methods that got generated from the attr_readers.